### DISCUSSION TITLE
Livolo TI0001-switch - I'm so confused

### DISCUSSION THREAD

**Discussion by nielsnieland2** [Koenkk/zigbee2mqtt#19121] [Devices]
I had some Aliexpress switches that didnt fit.. so my wife made me buy Livolo, its nice stuff. But I can't pair it with Zigbee2mqqtt.

My config yaml is as following: 
advanced:
  homeassistant_legacy_entity_attributes: false
  legacy_api: false
  legacy_availability_payload: false
  ext_pan_id: [33,117,141,25,0,75,18,0]
  channel: 26
  pan_id: 6756

I did every step from the page (https://www.zigbee2mqtt.io/devices/TI0001-switch.html) but when adding these values I get an 502  bad gateway and Zigbee2MQtt wont start.. Is there anyone who can help me?

**Comment by botany112** [comment]
did you got it working? its working here
